ts/lib/auth/googleauth.ts

```diff
@@ -105,6 +105,11 @@ export class GoogleAuth {
    */
   public static DefaultTransporter = DefaultTransporter;
 
+  /**
+   * @param {number=} refreshTokeEarlyMillis The token should be refreshed if it will expire within this many milliseconds.
+   */
+  constructor(private refreshTokenEarlyMillis?: number) {}
```

Comment:
Just curious - why is this private here, but public in the other classes?

Reply:
Switched to public here. There was no reason for this.